src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/DispatchRuntime.cs

```diff
@@ -213,19 +213,15 @@ public ServiceAuthorizationManager ServiceAuthorizationManager
             }
         }
 
-        public IAuthorizationService AuthorizationService
+        public IServiceScopeFactory ServiceScopeFactory
         {
```

Comment:
Public property on public class can't be removed.

Reply:
I kept the API public but marked it as Obsolete as I should have kept it internal in original PR.